R/pkg/R/functions.R

```diff
@@ -3071,6 +3085,19 @@ setMethod("array_position",
             column(jc)
           })
 
+#' @details
+#' \code{array_remove}: Removes all elements that equal to element from the given array.
+#'
+#' @rdname column_collection_functions
+#' @aliases array_remove array_remove,Column-method
+#' @note array_remove since 2.4.0
+setMethod("array_remove",
+          signature(x = "Column", value = "ANY"),
```

Comment:
should add in doc that for this function, `value` is ...?
https://github.com/huaxingao/spark/blob/87100ec766acf647e91ab14450f1c33c0f929d52/R/pkg/R/functions.R#L193